zulipterminal/core.py

```diff
@@ -622,15 +624,18 @@ def deregister_client(self) -> None:
         sys.exit(0)
 
     def exit_handler(self, signum: int, frame: Any) -> None:
-        question = urwid.Text(
-            ("bold", " Please confirm that you wish to exit Zulip-Terminal "),
-            "center",
-        )
-        popup_view = PopUpConfirmationView(
-            self, question, self.deregister_client, location="center"
-        )
-        self.loop.widget = popup_view
-        self.loop.run()
+        if self.exit_confirmation:
```

Comment:
This conditional appears to work absolutely fine :+1:

However, we could avoid needing to do conditional work and change this method, if we treat this method (and maybe rename it) as a 'prompting_exit_handler', and attach a minimal method as an 'no_prompt_exit_handler' if we do not require confirmation. That moves the conditional into the initializer, rather than having it down here.

Reply:
Thank you for your review, that makes sense.  I have incorporated the new approach and force-pushed the changes as suggested.